maven_iuvs/graphics/line_fit_plot.py

```diff
@@ -50,72 +146,56 @@ def detector_image(myfits, integration=0,
 
     cmap = idl_colorbars.getcmap(cmap)
 
-    ax.set_xlim([0, 1024])
-    ax.set_ylim([0, 1024])
-
-    # get the data
-    data = myfits['detector_dark_subtracted'].data[integration]
-
-    # figure out the binning
-    spapixlo = myfits['Binning'].data['SPAPIXLO'][0]
-    spapixhi = myfits['Binning'].data['SPAPIXHI'][0]
-    spepixlo = myfits['Binning'].data['SPEPIXLO'][0]
-    spepixhi = myfits['Binning'].data['SPEPIXHI'][0]
-    if not (set((spapixhi[:-1]+1)-spapixlo[1:]) == {0}
-            and set((spepixhi[:-1]+1)-spepixlo[1:]) == {0}):
-        raise ValueError("Error in bin table")
-
-    spepixrange = np.concatenate([[spepixlo[0]], spepixhi+1])
-    spapixrange = np.concatenate([[spapixlo[0]], spapixhi+1])
-
-    spepixwidth = spepixrange[1:]-spepixrange[:-1]
-    spapixwidth = spapixrange[1:]-spapixrange[:-1]
-
-    npixperbin = np.outer(spapixwidth, spepixwidth)
-
-    data = data/npixperbin
-
+    if plot_full_extent:
+        ax.set_xlim([0, 1024])
+        ax.set_ylim([0, 1024])
+    
     # figure out what norm to use
     if norm is None:
         if prange is None:
             prange = [0, 100]
         if arange is None:
-            arange = [np.percentile(data, prange[0]),
-                      np.percentile(data, prange[1])]
+            arange = [np.percentile(data_to_plot, prange[0]),
+                      np.percentile(data_to_plot, prange[1])]
+
+        # Allows overriding the vmin and vmax; for instance, if you 
+        # have multiple externally-created plots that you want to
+        # display with a shared colorbar.
+        vmin = force_vmin if force_vmin is not None else arange[0] 
```

Comment:
seems like arange and force_vmin,max are synonymous?

Reply:
Yes, must be an artifact, fixed by 07835d00fd301d96b8d508a211c52cee7710a049